packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -98,6 +111,26 @@ export class CanvasManager implements CanvasManagerInterface {
     this.locked = false;
   }
 
+  public snapshot(
+    dataURLOptions: DataURLOptions,
+    sampling: number,
+    blockClass: blockClass,
+    blockSelector: string | null,
+    unblockSelector: string | null,
```

Comment:
You could make these into a class member in the constructor (e.g. `this.options`), that way we don't always have to pass these in to `snapshot` (the other methods are fine to have these arguments). These options should not change between `snapshot()` calls and simplifies the public interface.